pkg/placement/placement.go

```diff
@@ -41,13 +42,20 @@ const (
 	// Once the first dissemination happens after getting leadership, membershipChangeWorker will
 	// use faultyHostDetectDefaultDuration.
 	faultyHostDetectInitialDuration = 6 * time.Second
-	faultyHostDetectDefaultDuration = 3 * time.Second
+	faultyHostDetectDefaultDuration = 4 * time.Second
```

Comment:
Is 1 second really that critical? what goes on during 1 second in terms of impact?

Reply:
I am reverting back to 3 seconds. I changed this when running the test in aks.. 

The implication of the duration change is that it will delay the operation to remove runtime from member list if there is OOMKill or intermittent network failure. This time duration gives the runtime to reconnect to placement so it will not change hashing table immediately. 

JFYI - If runtime exits gracefully (in upgrade or uninstall process), it will immediately remove the member from member list regardless of `faultyHostDetectDefaultDuration`. faultyHostDetectDefaultDuration is used only for unexpected stream disconnection.